datalad/cmdline/main.py

```diff
@@ -216,19 +216,22 @@ def main(args=None):
                 ret = cmdlineargs.func(cmdlineargs)
             except InsufficientArgumentsError as exc:
                 # if the func reports inappropriate usage, give help output
-                lgr.error('%s (%s)', exc_str_old(exc), exc.__class__.__name__)
+                ce = CapturedException(exc)
+                ce.log(lgr, logging.ERROR, "%s", ce.format_short())
```

Comment:
FWIW, I think we should aim for with  
```suggestion
                lgr.error("", exc_info=exc)
```
with all needed for logging as desired to happened by logger's Formatter

Reply:
or
```suggestion
                CapturedException(exc).log(lgr=lgr)
```
although thinking is needed about cases where custom msg etc is desired, then could be those
```
                CapturedException(exc).log(msg, *args, lgr=lgr)
```
so `.log` gets signature of logging's `_log` with extra `lgr`